index.rst

```diff
@@ -240,6 +240,101 @@ fitSrcMatchAstrometry
+- Convert source and object catalogs to Parquet before ever persisting them; only use SourceCatalog/FITS to hold Footprints.
+  We will retain the ``slot_*`` column names and not the underlying names they point to - downstream code should only be referring to the slot columns anyway, and in many cases we won't run multiple algorithms that could satisfy the slot.
+  In addition, note that the (final) ``source`` catalogs will be SDM-standardized before being written, so the the ``slot_*`` vs. underlying name question is moot there.
+
+Task and dataset type notes
+"""""""""""""""""""""""""""
+
+bootstrapImage
+   Initial background subtraction and detection of bright stars (galaxies are considered a nuisance here).
+   Initial versions of everything - astrometry, photometry, probably PSFs.
+   Probably aperture corrections of some kind, but targeted specifically at making compensated apertures work for FGCM.
+   These are all attached to its image output, ``initial_pvi``, which is a lot like today's ``calexp``.
+   This will be a fluence image with nJy pixel units, but how much of that is done by this task vs. ISR is TBD.
+   Its output catalog, ``initial_source_detector``, will be converted to Parquet before it is written, but not SDM-standardized.
+   Footprints will be written to a separate ``SourceCatalog`` dataset, ``initial_footprints``.
+   Whether this task does multiple detection rounds (to iterate on CR detection, the source detection filter, or the detection threshold) is TBD; we'd like to minimize that.
+   We would like to avoid running a deblender here (if we do, it will have to happen after we have the PSF model), but this depends on how this performs on crowded fields.
+   We are not thrilled with the name of this task and would love ideas for improvements (which should be coordinated with the names for ``bootstrapVisit``, ``finalizeImage``, ``compressImage``, and ``rebuildImage``).
+
+bootstrapVisit
+   Consolidate the per-detector outputs of ``bootstrapImage`` and recover from failures on some detectors by using those that succeeded (especially for WCSs).
+
+associateIsolatedStars
+   Pretty much just a renamed ``IsolatedStarAssociationTask``.
+
+solveAstrometry
+   Pretty much just a generic label for ``jointcal`` and ``GBDES``.
+
+fgcm
+   Nothing new here, except the sharding and names of the output datasets.
+
+modelVisitBackground
+   A replacement for SkyCorrectionTask, probably using PCA.
+   SkyCorrectionTask will serve as a placeholder until we have something better.
+   Also takes care of subtracting the wings of bright stars.
+
+finalizeCharacterization
+   Nothing new here except new names for the output datasets.
+
+   We are using "characterizations" here to mean "PSFs and aperture corrections", and are not thrilled about that, but we need something that means "PSFs and aperture corrections" that could also absorb aperture corrections being done rather differently than they are today.
```

Comment:
finalizeStelarProfileCorrections

Reply:
Not a bad idea, but at this point I'm just trying to get this merged and don't have energy to shepherd a naming decision.  Will punt.